src/Prettus/Repository/Criteria/RequestCriteria.php

```diff
@@ -46,7 +46,7 @@ public function apply($model, RepositoryInterface $repository)
 
         if ($search && is_array($fieldsSearchable) && count($fieldsSearchable)) {
 
-            $searchFields = is_array($searchFields) || is_null($searchFields) ? $searchFields : explode(';', $searchFields);
+            $searchFields = is_array($searchFields) || is_null($searchFields) ? $searchFields : array_filter(explode(';', $searchFields));
```

Comment:
actually i am not sure, but i think, that `array_filter()` also removes `false` and `0` values!

Reply:
Hey guys, I didn't have time to testing this PR. Can you merge? @johannesschobel  Did you try that, its removes falses and 0 values or not?

Reply:
@eullercdr if you preview this neat demo of `array_filter` here https://3v4l.org/eBOaY it shows, what is actually removed by `array_filter`. I have **not** tested it in the context of this specific PR - but from what i know from `array_filter` this could be very dangerous..

This _could probably_ (!) result in the fact, that you would not be able again to search for something that is `false` or has the value `''` (empty string) because it is removed from the search array!